testing/python/collect.py

```diff
@@ -1583,3 +1583,43 @@ def test_package_collection_infinite_recursion(testdir):
     testdir.copy_example("collect/package_infinite_recursion")
     result = testdir.runpytest()
     result.stdout.fnmatch_lines("*1 passed*")
+
+
+def test_package_with_modules(testdir):
+    """
```

Comment:
Just FYI, you could have created the actual files in `testing/example_scripts` and copied the whole directory here like the code above this function does.

The idea is that it is easier to execute the test using pytest as normal during testing/debugging. 😉 

This is fine as is, just thought I would mention this for the future. 👍